lib/beacon/broadcast.go

```diff
@@ -59,7 +58,7 @@ func writeBroadcasts(ctx context.Context, inbox <-chan []byte, port int) error {
 			addrs, err := intf.Addrs()
 			if err != nil {
 				l.Debugln(err)
-				return err
+				continue
```

Comment:
Interface discovery might work while retrieving the addresses doesn't. So log the error and carry on.

Reply:
Code comment instead of only GitHub?